test/jdk/javax/accessibility/manual/OptionPaneDemo.html

```diff
@@ -75,4 +75,4 @@
 </body>
 </html>
 
-                                            
\ No newline at end of file
+

```

Comment:
Ah, it looks the last line wasn't considered blank because it contained trailing spaces. There should be no trailing spaces, there should be only one blank line in the end of the file. In this case, the HTML file should have 77 lines.